SPARCore/GADArticle.m

```diff
@@ -1,7 +1,107 @@
 #import "GADArticle.h"
+#import "GADRemoteModel.h"
 
 @implementation GADArticle
 
+static NSString *const API_ARTICLE_ID = @"id";
+static NSString *const API_AUTHORS = @"authors";
+static NSString *const API_BRIEF = @"brief";
+static NSString *const API_CONTENT = @"content";
+static NSString *const API_DATE_PUBLISHED = @"datePublished";
+static NSString *const API_HEADER_IMAGE = @"headerImage";
+static NSString *const API_PUBLICATION_ID = @"publication";
+static NSString *const API_TITLE = @"title";
+
+static NSString *const API_HOSTNAME = @"https://g2j7qs2xs7.execute-api.us-west-2.amazonaws.com/";
+static NSString *const API_PREFIX = @"devstable";
+static NSString *const API_PUBLICATION_PATH = @"publications";
+static NSString *const API_ARTICLE_PATH = @"articles";
+
++ (void) articlesFromPublication: (NSString *)publicationId
+                                        completionHandler:(void(^_Nonnull)(NSArray<GADArticle *>
+                                                                           *_Nullable articles,
+                                                            NSError *_Nullable error))completion {
+    NSURL *queryURL = [GADArticle createURLWithPublication:publicationId];
+    
+    [GADRemoteModel fetchModelsWithParams:queryURL
+                          queryParameters:@{}
+                         modelTransformer:^(NSData *jsonData) {
+                            return [GADArticle articlesFromJSON:jsonData];
+                         }
+                        completionHandler:completion];
+
+}
+
+- (void) populateArticleWithId: (void(^_Nonnull)(GADRemoteModel * *_Nullable model,
+                                                NSError *_Nullable error))completion {
+    NSURL *queryURL = [GADArticle createURLWithArticle:self.articleId publication:self.publicationId];
+    
+    [GADRemoteModel fetchModelsWithParams:queryURL
+                          queryParameters:@{}
+                         modelTransformer:^(NSData *jsonData){
+                             return [GADArticle articlesFromJSON:jsonData];
+                         }
+                        completionHandler:^(NSArray <GADArticle *> *_Nullable model, NSError *_Nullable error){
+                            GADArticle *fullArticle = model[0];
```

Comment:
We may want to update other fields. It might make sense to write a function which will update a `GADArticle` with contents from another `GADArticle`. E.g.
```
- (void)updateWithArticle:(GADArticle *)newArticle {
  self.content = newArticle.content;
  self.authors = newArticle.authors;
  self.title = newArticle.title;
  ...
}
```
We'd want to include all fields that could conceivably change (`articleId` shouldn't change, though, etc.)